packages/opentelemetry-propagator-jaeger/package.json

```diff
@@ -0,0 +1,76 @@
+{
+  "name": "@opentelemetry/propagator-jaeger",
+  "version": "0.18.0",
+  "description": "OpenTelemetry Jaeger propagator provides HTTP header propagation for systems that are using Jaeger HTTP header format.",
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "repository": "open-telemetry/opentelemetry-js-contrib",
+  "scripts": {
+    "test": "nyc ts-mocha -p tsconfig.json 'test/**/*.test.ts' --exclude 'test/index-webpack.ts'",
+    "test:browser": "nyc karma start --single-run",
+    "tdd": "npm run tdd:node",
+    "tdd:node": "npm run test -- --watch-extensions ts --watch",
+    "tdd:browser": "karma start",
+    "codecov": "nyc report --reporter=json && codecov -f coverage/*.json -p ../../",
+    "clean": "rimraf build/*",
+    "lint": "eslint . --ext .ts",
+    "lint:fix": "eslint . --ext .ts --fix",
+    "precompile": "tsc --version",
+    "compile": "npm run version:update && tsc -p .",
```

Comment:
I think some of the script here need to be adapted/added/removed because the contrib repo uses a different build setup then the core repo.

Reply:
I cross checked and aligned with other packages. It looks to me like `*:browser` should be removed.